test/runnable/sdtor.d

```diff
@@ -4563,6 +4596,22 @@ void test18045() nothrow
 
 /**********************************/
 
+struct S66
+{
+    ~this() { }
+}
+
+nothrow void notthrow() { }
+
+class C66
```

Comment:
What is the purpose of this code? I don't see any instances of this class anywhere?

Reply:
> What is the purpose of this code?
To compile successfully.

>Maybe it should be moved to the compilable directory?
Perhaps, but I wanted the struct destructor tests in one file.